src/commonMain/kotlin/app/cash/turbine/flow.kt

```diff
@@ -92,10 +62,11 @@ public suspend fun <T> Flow<T>.test(
  * ```
  */
 public suspend fun <T> Flow<T>.test(
+  timeoutMs: Long? = null,
```

Comment:
The only public API should be `Duration`. We can delete the deprecated `Long` overloads completely now that 0.8 is shipped.